src/script/api/game_changelog.hpp

```diff
@@ -18,10 +18,16 @@
  * \b 1.9.0
  *
  * 1.9.0 is not yet released. The following changes are not set in stone yet.
+ * \li GSClientList
```

Comment:
This misses: `API additions:` on top of it, like all other versions. It is now a bit unclear under which category the well like this :)

Reply:
When touching this, I guess when adding new classes we also only mention the class, instead of listing individual methods.